lib/node_modules/@stdlib/lapack/base/dlarf1f/README.md

```diff
@@ -0,0 +1,324 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dlarf1f
+
+> Apply a real elementary reflector `H = I - tau * v * v ^ T` to a real M by N matrix `C`.
+
+<section class="intro">
+
+In linear algebra, a `Householder transformation` (or an `elementary reflector`) is a linear transformation that describes a reflection about a plane or a hyperplane containing the origin.
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var dlarf1f = require( '@stdlib/lapack/base/dlarf1f' );
+```
+
+#### dlarf1f( order, side, M, N, V, incv, tau, C, LDC, work )
+
+Applies a real elementary reflector `H = I - tau * v * v ^ T` to a real M by N matrix `C`.
+
+<!-- eslint-disable max-len -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var C = new Float64Array( [ 1.0, 5.0, 9.0, 2.0, 6.0, 10.0, 3.0, 7.0, 11.0, 4.0, 8.0, 12.0 ] );
+var V = new Float64Array( [ 0.5, 0.5, 0.5, 0.5 ] );
+var work = new Float64Array( 3 );
+
+var out = dlarf1f( 'row-major', 'left', 4, 3, V, 1, 1.0, C, 3, work );
+// returns <Float64Array>[ -4.5, -10.5, -16.5, -0.75, -1.75, -2.75, 0.25, -0.75, -1.75, 1.25,  0.25, -0.75 ]
+```
+
+The function has the following parameters:
+
+-   **order**: storage layout.
+-   **side**: specifies the side of multiplication with `C`. Use `left` to form `H * C` and `right` to from `C * H`.
+-   **M**: number of rows in `C`.
+-   **N**: number of columns in `C`.
+-   **V**: the vector `v` in the representation of `H` as a [`Float64Array`][mdn-float64array].
+-   **incv**: stride length for `V`. If `incv` is negative, the elements of `V` are accessed in reverse order.
+-   **tau**: the value of `tau` in representation of `H`.
+-   **C**: input matrix stored in linear memory as a [`Float64Array`][mdn-float64array].
+-   **LDC**: stride of the first dimension of `C` (a.k.a., leading dimension of the matrix `C`).
+-   **work**: workspace [`Float64Array`][mdn-float64array].
+
+Based on the `side` of multiplication, the expected storage of the array changes:
+
+-   `work` should have `N` indexed elements if `side` = `left` and `M` indexed elements if `side` = `right`.
+-   `V` should have `1 + (M-1) * abs(incv)` indexed elements if `side` = `left` and `1 + (N-1) * abs(incv)` indexed elements if `side` = `right`.
+
+The sign of the increment parameter `incv` determines the order in which elements of `V` are accessed. For example, to access elements in reverse order,
+
+<!-- eslint-disable max-len -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var C = new Float64Array( [ 1.0, 5.0, 9.0, 2.0, 6.0, 10.0, 3.0, 7.0, 11.0, 4.0, 8.0, 12.0 ] );
+var V = new Float64Array( [ 0.5, 0.5, 0.5, 0.5 ] );
+var work = new Float64Array( 3 );
+
+var out = dlarf1f( 'row-major', 'left', 4, 3, V, -1, 1.0, C, 3, work );
+// returns <Float64Array>[ -4.5, -10.5, -16.5, -0.75, -1.75, -2.75, 0.25, -0.75, -1.75, 1.25,  0.25, -0.75 ]
```

Comment:
This example is not super informative, as `V` is symmetric. Whether you go forward or reverse, you get the same values. I suggest updating this example to make it more meaningful.